### PR TITLE
First attempt to "fix" #179

### PR DESCRIPTION
**Pull request by remcovanvugt**
This PR serves to illustratie issue #179 
I don't know the internals of Spiegel any further, so my judgement on how this should actually be fixed is limited.

### CODE REVIEW

File: src/update-listeners.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const Globals = require('./globals')
 const log = require('./log')
 const sporks = require('sporks')
 const utils = require('./utils')
+const urlsafe = require('querystring').escape
```
Comment (redgeoff) [inline src/update-listeners.js]: To be consistent with naming conventions, please rename this to `urlSafe`

**Comment by redgeoff** [comment]
Thanks @remcovanvugt this looks good on the surface. Could you please fix the comment in the review and then create a test for _toDBName() in https://github.com/redgeoff/spiegel/blob/master/test/spec/update-listeners.js for both the case where there are slashes and are not slashes in the name?

**Comment by remcovanvugt** [comment]
Thanks @redgeoff I tried my best at creating a suitable test, but this a bit far from my field of knowledge so please excuse my bad code ;)
So far it seems to work, the test passes, and the fix works as intended as well in my test setup.

File: test/spec/update-listeners.js [hunk 2]
```diff
@@ -105,7 +105,7 @@ describe('update-listeners', () => {
   }
 
   const createTestDBs = async() => {
-    await testUtils.createTestDBs(['test_db1' + suffix, 'test_db3' + suffix])
+    await testUtils.createTestDBs(['test_db1' + suffix, 'test_db3' + suffix, 'account/1/2'])
```
Comment (redgeoff) [inline test/spec/update-listeners.js]: I don't think this change is needed for your test. Please remove it

File: test/spec/update-listeners.js [hunk 3]
```diff
@@ -303,6 +303,23 @@ describe('update-listeners', () => {
     })
   })
 
+  it('should handle dbnames with slashes', async() => {
+    await createListeners()
```
Comment (redgeoff) [inline test/spec/update-listeners.js]: This shouldn't be needed

File: test/spec/update-listeners.js [hunk 4]
```diff
@@ -303,6 +303,23 @@ describe('update-listeners', () => {
...
+
+    let DBNameWithSlashes = {}
```
Comment (redgeoff) [inline test/spec/update-listeners.js]: please rename the variables to `dbNameWithSlashes` and `dbNameWithoutSlashes`

File: test/spec/update-listeners.js (hunk 2 above)
Comment (remcovanvugt) [reply test/spec/update-listeners.js]: Sorry, forgot about that line. I removed it.

File: test/spec/update-listeners.js (hunk 3 above)
Comment (remcovanvugt) [reply test/spec/update-listeners.js]: Removed this one as well.

File: test/spec/update-listeners.js (hunk 4 above)
Comment (remcovanvugt) [reply test/spec/update-listeners.js]: Done.

**Comment by redgeoff** [comment]
Thanks @remcovanvugt!

**Comment by redgeoff** [comment]
Just deployed 1.1.0 with the changes

**Pull request merged**